src/transactions/feed/queryHelper.ts

```diff
@@ -228,30 +331,44 @@ async function queryTransactionsFeed(
     },
     body: JSON.stringify({
       query: TRANSACTIONS_QUERY,
-      variables: { address, localCurrencyCode, afterCursor },
+      variables: { address, localCurrencyCode, chain, afterCursor },
```

Comment:
I think it's fine to use the Chain enum you defined throughout most of the app, but at the API level I think we should use chain ID's agreed on here https://valora-app.slack.com/archives/C025V1D6F3J/p1689089907854609

Reply:
I realize this will require an update in blockchain-api but I think it's worth it

Reply:
I made an attempt and converting between testnet/mainnet-agnostic "chain names" and "chain id"'s [here](https://github.com/valora-inc/wallet/pull/4099/files) that you may find useful

Reply:
in particular you can look [here](https://github.com/valora-inc/wallet/blob/4e6491825cba38d209e702f70f9e8573e98b7acd/src/utils/blockchains.ts) and [here](https://github.com/valora-inc/wallet/blob/4e6491825cba38d209e702f70f9e8573e98b7acd/src/config.ts#L103)

Reply:
another way to do it could be to define chain ID's as new configuration parameters. Though I avoided it because I felt like it introduced a potential for bugs where your eth chain id and your celo chain id don't match (one is mainnet and one is testnet, for instance)

Reply:
Ah, interesting -- yeah, I'll add this into this PR, thanks for the references here.

Reply:
Actually, this raises an interesting point -- blockchain-api already has deployments in mainnet and testnet envs, and thus we don't actually need to specify the _specific_ blockchain environment when parameterizing by chain in GraphQL calls. Our "formal" chain specifiers are environment specific, however. Updating blockchain-api to require us to use these environment-specific identifiers might be a little weird. For instance, when typing the request schema, what should the type of `chain` be? It seemingly depends on the environment it's deployed in, unless we want e.g. `"ethereum-goerli"` to be a statically valid (but semantically meaningless) chain to request against the mainnet blockchain-api deployment (which would presumably just throw at runtime).

Reply:
in my draft PR I suggested we use a test/mainnet-agnostic "chain name" enum that then maps to "chain id" depending on the env vars when it accesses the API's of various backend services. one option for dealing with chain id's in the backend services themselves is to do essentially the same thing in reverse, mapping chain ID's to chain names. We could throw errors if we get the "wrong" chain ID for the deployment of the service, e.g. a dev server receives a mainnet request. The upside of this is if we mis-configured our development/production app to hit the wrong backend, it would be less likely to silently failed (unless we also mis-configured which blockchain network to use, testnet vs mainnet). The downside is it's a bit more complex than just using chain names, for limited gains-- I'm not sure we've ever seen this kind of configuration error in the past.

Other alternatives are:
1. accept testnet or mainnet requests from all deployments (seems pointless to have multiple deployments at that point)
2. ditch the chain ID idea and use test/mainnet-agnostic chain names at the API level

I think this probably warrants discussion with the broader eng team